modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectEntryDocumentContributor.java

```diff
@@ -67,16 +67,19 @@ private void _setFolder(Document document, long objectEntryFolderId) {
 				objectEntryFolderId);
 
 		if (objectEntryFolder == null) {
+			document.addKeyword("cms_section", "none");
```

Comment:
avoid to add extra index attributes for non CMS use cases, in this way everyone that today is using Object will find a new attribute even when they are not using CMS